practice-project-api/routes/calculateRoutes.ts

```diff
@@ -9,33 +10,46 @@ enum CalculatorOperation {
   Subtract = '-'
 }
 
-calculateRoute.post("/calculate", (req: Request, res: Response, next) => {
+calculateRoute.post(
+  "/calculate",
+  body('inputs.firstValue', 'please enter a number').not().isEmpty().matches(/^-?\d*\.?\d+$/),
+  body('inputs.secondValue', 'please enter a number').not().isEmpty().matches(/^-?\d*\.?\d+$/),
+  body('operation', 'please select one of the operations from the dropdown').not().isEmpty().isIn(['+', '-', '*', '/']),
```

Comment:
you could probably use the `CalculatorOperation` enum when checking

Reply:
use something like `Object.values(CalculatorOperation)` to get the values in an array and then you can use that instead of hard coding